ci/scripts/deterministic-recovery-test.sh

```diff
@@ -44,7 +44,7 @@ filter_stack_trace_for_all_logs() {
   done
 }
 
-trap filter_stack_trace_for_all_logs ERR
+# trap filter_stack_trace_for_all_logs ERR
```

Comment:
Why?

Reply:
`filter_stack_trace_for_all_logs` accidentally removes some useful stacktrace from madsim.

But without it, the stacktrace dump in the log file is really large, and makes the file size few hundred MB to 1GB+ previously.

So I temporarily disable it here. This change won't be merged in.